modules/prometheus-exporter_oracledb/conf/readme.yaml

```diff
@@ -0,0 +1,83 @@
+documentations:
+  - name: Prometheus Exporter for oracledb
+    url: https://github.com/iamseth/oracledb_exporter
+
+source_doc: |
+  The detectors of this module uses defaults metrics from the [oracledb exporter] (https://github.com/iamseth/oracledb_exporter) and optionnaly custom metrics from custom templates which have to be specified during the deployment.  
```

Comment:
```suggestion
  The detectors of this module uses defaults metrics from the [oracledb exporter](https://github.com/iamseth/oracledb_exporter).
```

Reply:
yes, done ..

Reply:
not yet, commit the suggestion directly would preserve you of copy/paste error.